cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/DRep.hs

```
@@ -236,16 +267,17 @@ hprop_golden_governance_drep_registration_certificate_id_hex = propertyOnce . H.
     , "--key-reg-deposit-amt", "0"
     , "--drep-metadata-url", "dummy-url"
     , "--drep-metadata-hash", "52e69500a92d80f2126c836a4903dc582006709f004cf7a28ed648f732dff8d2"
+    , "--trust-drep-metadata-hash"
```

Not checking the hash should be a silent default. You need to opt into this behaviour.

---

Based on [this discussion](https://x.com/carloslodelar/status/1833506267102400763), it was my understanding that this was the consensus. Do you mean we should do it differently in this particular case?

---

Having it in the help options is enough. It's unnecessary to make users explicitly opt out.

---

Got it. I initially followed our earlier discussion and still think that approach made sense, but I’m fine with changing it. Let me know if you want to chat more, or I’ll go ahead with the updates

---

Addressed here: https://github.com/IntersectMBO/cardano-cli/commit/baafefd637584bcf2faf7d8549237f7718b364a9